workers/unity/Packages/io.improbable.gdk.core/Worker/MessagesToSend.cs

```diff
@@ -20,8 +26,8 @@ public class MessagesToSend
 
         private readonly List<IComponentDiffStorage> componentStorageList = new List<IComponentDiffStorage>();
 
-        private readonly Dictionary<uint, Dictionary<uint, IComponentCommandSendStorage>> componentIdToCommandIdToStorage =
-            new Dictionary<uint, Dictionary<uint, IComponentCommandSendStorage>>();
+        private readonly Dictionary<(uint componentId, uint commandId), IComponentCommandSendStorage> componentCommandToStorage =
```

Comment:
What are the rules for `GetHashCode()` on tuples?

Reply:
It will generate the hash for both and combine them using hashcode.combine.
Suitable for indexing.

Reply:
Perfect 💯